terraform/README.md

```diff
@@ -1,56 +1,91 @@
 # Terraform Provider Plugin
 
-# Installation
+## Usage
 
-1. Clone the plugin:
+Please, refer to [GETTING_STARTED guide](GETTING_STARTED.md) and [official documentation](https://goteleport.com/docs/setup/guides/terraform-provider/).
```

Comment:
I would be in favor of dropping the getting started guide and just referencing the official documentation instead. They share the same information, but keeping both is just additional upkeep. Up to you.